react-graphql-apollo-tutorial.md

```diff
@@ -980,7 +982,7 @@ const STAR_REPOSITORY = gql`
 ...
 {{< /highlight >}}
 
-The mutation definition takes the `id` variable as input for the actual `addStar` mutation. Furthermore, as you have done before, you can decide what should be returned in case of a successful mutation. Now, you can use a Mutation component analog to the previously used Query component. You have to pass the mutation prop, but this time also a variable prop for passing the identifier for the repository.
+The mutation definition takes the `id` variable as input for the `addStar` mutation. As before, you can decide what should be returned in case of a successful mutation. Now, you can use a Mutation component the represents the previously used Query component. You have to pass the mutation prop, but also a variable prop for passing the identifier for the repository.
```

Comment:
"Now, you can use a Mutation component the represents the previously used Query component." to "Now, you can use a Mutation component that represents the previously used Query component, but this time for 
 mutations."